### ISSUE TITLE
equalizer within equalizer

### ISSUE THREAD

**Issue by cortopy** [zurb/foundation#6279]
I currently have two "equalized" columns. In the right column there is a second equalizer for a block grid list. 

My problem is that the equalizer within the right column looks like is being fired after the first one, thus making the block grid overflow the right column. Once the first equalizer has set the height for the two main columns, I'm unable to fire it again.

At the moment my only solution to this problem is using this function

```
$(document).foundation({
  equalizer: {      
    after_height_change: function(){
      var height = Math.max($("#left").height(), $("#main-right").height()); 
        $("#left").height(height);
        $("#main-right").height(height);
    }
  }
});
```

Basically what this acts as a substitute for the first equalizer. This way there is no problem in the order in which functions are triggered.

The problem with this solution is that this will be fired after every equalizer on the site regardless of whether there are left/right columns. And even worse, this is incompatible with using a first equalizer. 

Using the "reflow" method  described in the doc instead of the Math.max used here would create a bad loop and break the page. So that's not possible.

Is there anyone with the same problem who has found a different alternative?

**Comment by rafibomb** [comment]
Can we see the markup for this example?

**Comment by cortopy** [comment]
Thanks a lot @rafibomb for looking into this

I'm afraid I'm still developing site on localhost but I've created a fiddle with a similar markup to the site. My only problem is that for some reason I'm not able to make equalizer work on the fiddle (the rest of foundation seems to work fine)

The markup in the fiddle is the way I initially had it. That is without the Math.max JS and with this structure:

1. Equalizer one
2.  Equalizer two with block grid

https://jsfiddle.net/cortopy/ncmoL9vj/4/

**Comment by jeaniec** [comment]
@cortopy  We just recently merged in a pull request that accounts for grouped Equalizer elements as of https://github.com/zurb/foundation/commit/5c537d14ea6c6e5918c9d7060cdf8c8d06ecf493. Let us know if that works for your use case.

**Comment by cortopy** [comment]
Thanks for that @jeaniec But I'm afraid it's not solved the problem. 

I'm afraid the new version is actually making things even worse for my case. Before the changes, the nested equalizer would be fired after the second one and so nested contents would overflow if higher than the height set beforehand for parents. Now the "parent" equalizer is not fired at all, neither before nor after the nested equalizer. So the result is a very small left section.

I'm going back to the older version as at least it works with my fiddly workaround. Sorry

**Comment by jeaniec** [comment]
Thanks @cortopy. We'll file this particular use case away in our next iteration for Equalizer 6. Thanks for fleshing this out with us!
